common-app-lib/src/cmr/common_app/site/data.clj

```diff
@@ -31,7 +31,8 @@
   []
   (merge (time-page)
          default-partner-guide
-         {:base-url "../../../"}))
+         {:base-url "../../../"
+          :status-app-url (config/status-app-url)}))
```

Comment:
Only comment would be to possibly have the status app also be optional in the case of it not being set
```
(merge (time-page)
            default-partner-guide
            (when-not (string/blank? (config/status-app-url)) (config/status-app-url)))
```

Reply:
The defconfig status-app-url is defaulting to the prod status app url